### DISCUSSION TITLE
The April 2023 Reunion

### DISCUSSION THREAD

**Discussion by si** [multipack/website#48] [General]
Following a few messages on the Twitter group DM and [Mastodon](https://managingengineers.net/@si/109949557745663333), we've settled on a date for a much-needed reunion back in Birmingham.

## Details 

Date: **Thursday 6th April 2023**
Time: **6pm-9pm**
Venue: **The Old Joint Stock, Birmingham**

[Details are on the website](https://multipack.co.uk/events/2023/reunion/) (thanks to @trovster for helping publish)

## Who's in? 

Add your RSVP in the thread _below by Monday 3rd April_ so I can try to reserve a table.

**Comment by paulrobertlloyd** [comment]
Corrected the venue name 😏 but should also remove the year from the URL slug as its superfluous (am on a train right now, and GitHub's iOS app only lets you do so much)

**Reply by trovster** [reply]
I have changed the slug to "reunion": https://multipack.co.uk/events/2023/reunion/
